server/src/main/java/org/apache/druid/metadata/SQLMetadataConnector.java

```diff
@@ -327,6 +331,29 @@ tableName, getPayloadType()
         )
     );
   }
+  
+  public boolean tableContainsColumn(Handle handle, String table, String column)
```

Comment:
No, modified access

Reply:
Still seems to be public.

Reply:
Sorry, I had modified access for a different method.
This method needs to be public since it needs to be overridden and used for derby in a different package